roles/cifmw_cephadm/templates/cert_ceph_rgw.yml.j2

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ cifmw_cephadm_cert_name }}
+  namespace: {{ cifmw_cephadm_ns }}
+spec:
+  duration: {{ cifmw_cephadm_cert_issuer_duration }}
+  issuerRef: {{ cifmw_cephadm_cert_issuer_ref }}
+  secretName: {{ cifmw_cephadm_cert_name }}
+  dnsNames:
```

Comment:
Update like this:
```yaml
dnsNames:
    - {{ cephadm_vip }}
{% for host in _hosts %}
  - {{ host }}
{% endfor %}
```

Reply:
If you don't do this, then opentack client pod will get this when it talks to haproxy:
```
2024-06-18 14:29:46,783 28 WARNING  [urllib3.connection] Certificate did not match expected hostname: 192.168.122.2. Certificate: {'subject': (), 'issuer': ((('commonName', 'rootca-public'),),), 'version': 3, 'serialNumber': 'A36019DFBEC55F5C04EAF00C07847505', 'notBefore': 'Jun 14 16:38:40 2024 GMT', 'notAfter': 'Jun 13 16:38:40 2029 GMT', 'subjectAltName': (('DNS', 'compute-0'), ('DNS', 'compute-1'), ('DNS', 'compute-2'))}
```